### PR TITLE
Hateffect minor update

### PR DESCRIPTION
**Pull request by admkakaroto**
Just a little sync with jRO, kRO and twRO.

* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/8121

* **Server Mode**: RE / PRE-RE

* **Description of Pull Request**: Just a little hateffect update

### CODE REVIEW

**Review by aleos89 — APPROVED** [review]

File: src/map/script.hpp [hunk 1]
```diff
@@ -2123,6 +2123,10 @@ enum e_hat_effects : int16{
 	HAT_EF_C_ROS2023_CAPE_1,
```
Comment (Pokye) [inline src/map/script.hpp:2123]:
```suggestion
	HAT_EF_C_ROS2023_CAPE_1,
	HAT_EF_BLACK_THUNDER_,
```

File: src/map/script_constants.hpp [hunk 2]
```diff
@@ -9624,6 +9624,10 @@
 	export_constant(HAT_EF_C_ROS2023_CAPE_1);
```
Comment (Pokye) [inline src/map/script_constants.hpp:9624]:
```suggestion
	export_constant(HAT_EF_C_ROS2023_CAPE_1);
	export_constant(HAT_EF_BLACK_THUNDER_);
```

**Review by Pokye — COMMENTED** [review]
In kRO's hateffectinfo there is an effect with the same name, but with a different ID and effect, in mine I used it this way.
image

File: src/map/script_constants.hpp (hunk 2 above)
Comment (admkakaroto) [reply src/map/script_constants.hpp:9624]: Oh yes, i have not checked this, thanks!

**Pull request merged**